src/views/Sales/SaleDetails.vue

```diff
@@ -24,6 +24,12 @@
         <template v-slot:item.owner="{ item }">
           <user-tool-tip :user="item.owner" @click.stop />
         </template>
+
+        <template v-slot:item.return="{ item }">
+          <v-btn variant="plain" @click.stop="submitReturn(item)">
+            <v-icon size="25">mdi-backup-restore</v-icon>
```

Comment:
I don't like the icon so I went ahead and looked at some. Should we go with cart-remove, cart-arrow-up or... cash-remove, or cash-refund. Take a look. I couldn't decide among these. Take your pick

Reply:
cart-remove looks the most suitable for me